### PR TITLE
v21, v22: back to xfs

### PR DESCRIPTION
**Pull request by kopiczko**
Fixes https://github.com/giantswarm/giantswarm/issues/5730

Switching back to XFS before 8.0.0 is active. We couldn't reproduce XFS issue on CoreOS 2023.5.0 while it still occurs on 2023.4.0.

### CODE REVIEW

File: service/controller/v21/cloudconfig/master_template.go [hunk 1]
```diff
@@ -164,7 +164,7 @@ Description=Mount for docker volume
 [Mount]
 What=/dev/disk/by-id/virtio-dockerfs
 Where=/var/lib/docker
-Type=ext4
+Type=xfs
```
Comment (kopiczko) [inline service/controller/v21/cloudconfig/master_template.go]:
```suggestion
Type=ext4
```

File: service/controller/v21/cloudconfig/master_template.go [hunk 2]
```diff
@@ -178,7 +178,7 @@ Description=Mount for kubelet volume
 [Mount]
 What=/dev/disk/by-id/virtio-kubeletfs
 Where=/var/lib/kubelet
-Type=ext4
+Type=xfs
```
Comment (kopiczko) [inline service/controller/v21/cloudconfig/master_template.go]:
```suggestion
Type=ext4
```

File: service/controller/v21/cloudconfig/worker_template.go [hunk 3]
```diff
@@ -129,7 +129,7 @@ Description=Mount for docker volume
 [Mount]
 What=/dev/disk/by-id/virtio-dockerfs
 Where=/var/lib/docker
-Type=ext4
+Type=xfs
```
Comment (kopiczko) [inline service/controller/v21/cloudconfig/worker_template.go]:
```suggestion
Type=ext4
```

File: service/controller/v21/cloudconfig/worker_template.go [hunk 4]
```diff
@@ -143,7 +143,7 @@ Description=Mount for kubelet volume
 [Mount]
 What=/dev/disk/by-id/virtio-kubeletfs
 Where=/var/lib/kubelet
-Type=ext4
+Type=xfs
```
Comment (kopiczko) [inline service/controller/v21/cloudconfig/worker_template.go]:
```suggestion
Type=ext4
```

File: service/controller/v21/key/key.go [hunk 5]
```diff
@@ -54,7 +54,7 @@ const (
 	CoreosVersion        = "2023.5.0"
 
 	K8SEndpointUpdaterDocker = "quay.io/giantswarm/k8s-endpoint-updater:590479a6228c2c143695a268bda5382b52f7ffe1"
-	K8SKVMDockerImage        = "quay.io/giantswarm/k8s-kvm:6fbba190dbdab2a24dd4022f49510575ad482aaa"
+	K8SKVMDockerImage        = "quay.io/giantswarm/k8s-kvm:1d9e3a72877598353c0f6da5680db55b8ead108b"
```
Comment (kopiczko) [inline service/controller/v21/key/key.go]:
```suggestion
	K8SKVMDockerImage        = "quay.io/giantswarm/k8s-kvm: 6fbba190dbdab2a24dd4022f49510575ad482aaa"
```

File: service/controller/v21/key/key.go [hunk 6]
```diff
@@ -54,7 +54,7 @@ const (
 	CoreosVersion        = "2023.5.0"
 
 	K8SEndpointUpdaterDocker = "quay.io/giantswarm/k8s-endpoint-updater:590479a6228c2c143695a268bda5382b52f7ffe1"
-	K8SKVMDockerImage        = "quay.io/giantswarm/k8s-kvm:6fbba190dbdab2a24dd4022f49510575ad482aaa"
+	K8SKVMDockerImage        = "quay.io/giantswarm/k8s-kvm: 6fbba190dbdab2a24dd4022f49510575ad482aaa"
```
Comment (kopiczko) [inline service/controller/v21/key/key.go]:
```suggestion
	K8SKVMDockerImage        = "quay.io/giantswarm/k8s-kvm:6fbba190dbdab2a24dd4022f49510575ad482aaa"
```

**Review by tuommaki — APPROVED** [review]
:+1:

**Comment by mazzy89** [comment]
I assume we should include this in the last version 8.0.0. which is the status?

**Review by mazzy89 — COMMENTED** [review]
LGTMITWFY

**Comment by kopiczko** [comment]
> I assume we should include this in the last version 8.0.0. which is the status?

Yes we'd like to have it in 8.0.0 very much.

**Comment by mazzy89** [comment]
Waiting for this one to be kicked in and then we can release 8.0.0

**Comment by kopiczko** [comment]
We also need to add changelogs. I'll do that in separate PR because I'm testing this one.

**Comment by mazzy89** [comment]
Sure. I'll also need to add it in the Release Notes for 8.0.0

**Pull request merged**